tools/winmd2objc/lib/libwinmd2objc.vcxproj

```diff
@@ -0,0 +1,214 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\packages\NuGet.Build.Packaging.0.1.253\build\NuGet.Build.Packaging.props" Condition="Exists('..\..\..\packages\NuGet.Build.Packaging.0.1.253\build\NuGet.Build.Packaging.props')" />
```

Comment:
Yep, this was through visual studio.

---
In reply to: [117095345](https://github.com/Microsoft/WinObjC/pull/2708#discussion_r117095345) [](ancestors = 117095345)